api_test/__init__.py

```diff
@@ -26,7 +26,7 @@
 
 class Plugin(QueryHandler):
     def id(self):
-        return "someid";
+        return "test";
```

Comment:
Usually, plugin use `__name__`.
`name(self)` is `md_name`, and so on.
Do you think this test plugin should diverge from that?

Copied from other plugin:
```
    def id(self):
        return __name__

    def name(self):
        return md_name

    def description(self):
        return md_description

```

Reply:
Note that __name__ is albert.__name__ for plugins. That's a design decision to avoid name conflicts, e.g. that docker does not import itself. md_name is the module name if you don't override it.